azure/terraform/modules/app_service/variable.tf

```diff
@@ -49,6 +49,11 @@ variable "bypass_dfe_sign_in" {
   description = "Bypass DFE Sign-In authentication and use a default role"
 }
 
+variable "PR_NUMBER" {
+  type        = string
+  description = "PR number from Azure Devops"
```

Comment:
"Github pull request number provided by Azure Devops" ?

Reply:
its stored in the azure devops variable $(System.PullRequest.PullRequestNumber)
 this is where i am transferring the value to terraform from